streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/domain/RestResponse.java

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.streampark.console.base.domain;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
this dependency (maven pom) is not imported in any pom, in other words, can it still be compiled in a non intellij idae?